### ISSUE TITLE
*OTHER* Care Home Residency 

### ISSUE THREAD

**Issue by annaschultze** [opensafely/codelist-development#64]
Care Home Residency can be identified by either Read or SNOMED codes. Previous Read code lists are published (Shah et al, 2010).  

There are currently two SNOMED codes which have already been created as a code list (https://codelists.opensafely.org/codelist/opensafely/nhs-england-care-homes-residential-status-ctv3/1e037bde/). We should consider either adding other CTV3 codes or creating a separate varaible with these. I've created a draft CTV3 list here:  https://codelists.opensafely.org/builder/60b18755/ for review by searching for terms "care" and "nursing" and selecting relevant codes. 

My questions are: 

- Is there an easy way to translate prior Read code lists to see whether we've missed any codes? 
- Should we include Read codes for temporary residents or not (Ive currently included codes for temporary residents, but not past residents)

**Comment by brianmackenna** [comment]
> Is there an easy way to translate prior Read code lists to see whether we've missed any codes?

@inglesp can you help with this translation from Shat et al (@AnnaB45 presume you have a file with all codes) - should we be perhaps translating straight to Snomed?

> Should we include Read codes for temporary residents or not (Ive currently included codes for temporary residents, but not past residents)

I think this depends on how we specify time of exposure. If we are looking at a very specific time e.g. this week in february or ever lived in a care home, i think they should be included.

**Comment by annaschultze** [comment]
Wrong Anna but I've got it :) Shah shared their codes but as a pdf (attached), and the codes are indicated as a list rather than as individual codes. If @inglesp could advise on the best way to make that usable that would be good. I unfortunately don't have access to a Read code brower at the moment. 
[afq086_suppl.pdf](https://github.com/opensafely/codelist-development/files/5903926/afq086_suppl.pdf)


On the second point - I think we should consider just using the diagnostic codes over a certain time anyway (like a year), so I'll include for now and we can mention in the docs this is how the codelist works.

**Comment by annaschultze** [comment]
Other codes to add from Brian:  https://codelists.opensafely.org/codelist/primis-covid19-vacc-uptake/longres_cod/v1/#full-list (should remove children's unit and children's home)

**Comment by inglesp** [comment]
Here you go.  (You'll want to change the file extensions... GH won't let me upload CSV files.)

`carehomes-snomedct.csv` contains the SNOMED codes.  You should check all these, and make sure that you're happy with them.  The `is_assured` column tells us whether the SNOMED code came via an "assured" mapping.  SNOMED codes that come via unassured mappings are less likely to be what you want than those that come via assured mappings.

`carehomes-mapping.csv` contains the mappings that were used.  You may not need this, but if you want to check why a given SNOMED code is in `carehomes-snomedct.csv`, this should tell you.

[carehomes-snomedct.csv.txt](https://github.com/opensafely/codelist-development/files/5911943/carehomes-snomedct.csv.txt)
[carehomes-mapping.csv.txt](https://github.com/opensafely/codelist-development/files/5911942/carehomes-mapping.csv.txt)

**Comment by annaschultze** [comment]
Ok - I've discussed with Peter and it's difficult to extract the code information from the pdf. For now, we will therefore proceed using the code list builder. I've updated this list here : https://codelists.opensafely.org/builder/60b18755/?filter=included , including PRIMIS codes but excluding codes specifically relating to children's residential care (children's home and welfare house). I've currently retained unspecified resedential care even though this could lead to missclassification, as I will be applying an age restriction to the care home population. I'll include this in the methodology. 

@brianmackenna would you be happy to do clinical sign off? I know we can use the PRIMIS list but there are quite a few additional codes here, and think worth investigating the potential extra coverage we could gain.

**Comment by brianmackenna** [comment]
Agreed with @annaschultze  For the purposes of current work we will use the COVID call and recall definition of care home codes and NHSE codes.

^we will park this [longer list](https://codelists.opensafely.org/builder/60b18755/?filter=included) of codes and return to it at a later date for assessment of usefulness.
